src/python/pants/backend/terraform/tool.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.core.util_rules.external_tool import TemplatedExternalTool
+from pants.engine.rules import collect_rules
+from pants.util.meta import classproperty
+
+
+class TerraformTool(TemplatedExternalTool):
+    options_scope = "download-terraform"
+    name = "terraform"
+    help = "Terraform (https://terraform.io)"
+
+    default_version = "0.14.5"
```

Comment:
We should remember to document how to update this in config, because people may do so aggressively.

Or should we go ahead with that follow-up of allowing you to not set this and falling back to a terraform binary on the PATH.

Reply:
> Or should we go ahead with that follow-up of allowing you to not set this and falling back to a terraform binary on the PATH.

That won't work with remote execution though, correct?

Reply:
> We should remember to document how to update this in config, because people may do so aggressively.

In the readme.io docs site?

Reply:
I also need to bump this to Terraform v1.0. Will do that in an immediate follow-up.

Reply:
It will if there is a `terraform` binary on the RE image.